powerhome/playbook
# [PLAY-460] Website: kit pages in React

#### Screens

<img width="1344" alt="Screen Shot 2022-12-01 at 10 01 13 AM" src="https://user-images.githubusercontent.com/73671109/205087344-ab4adac3-75db-4a5c-aa14-588e4cf1949b.png">

#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-460)

#### How to test this

In the url type in /kits/:name/react/new OR /kits/:name/rails/new and check to see that the correct kit name is being rendered as a title

#### Checklist:
